.github/actions/ci-setup/action.yaml

```diff
@@ -5,7 +5,7 @@ inputs:
     default: 16.15
   pnpm-version:
     description: "PNPM version"
-    default: 7.5.1
+    default: 7.8.0
```

Comment:
You could use `default: latest` instead maybe, no?!

Reply:
I think it's better to keep specific version because
- setups should be as stable as possible
- we're using pnpm version in cache key for CI 
```
key: ${{ runner.os }}-node-${{ inputs.node-version }}-pnpm-${{ inputs.pnpm-version }}-lock-${{ hashFiles('**/pnpm-lock.yaml') }}
```

We don't wanna unexpected problems out from nothing, sometimes spending time to debug and understand that the problems was with `pnpm` version specifically. 
From times to times we should upgrade it to easier identify breaking changes if there are any